src/masternode-sync.cpp

```diff
@@ -83,7 +83,7 @@ void CMasternodeSync::Reset()
 void CMasternodeSync::AddedMasternodeList(uint256 hash)
 {
     if(mnodeman.mapSeenMasternodeBroadcast.count(hash)) {
-        if(mapSeenSyncMNB[hash] < MASTERNODE_SYNC_THRESHOLD) {
+        if(mapSeenSyncMNB.count(hash) && mapSeenSyncMNB[hash] < MASTERNODE_SYNC_THRESHOLD) {
```

Comment:
By the way, I have a living machine where the carsh always occurs, It is a win32. Do you want me to report something?